components/List/list.html

```diff
@@ -0,0 +1,113 @@
+            style="width: 800px; height: 285px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+          </iframe>
+
+    <section id="list">
+        <h2 class="pb">Spaced Lists</h2>
+        <div class="list-container">
+        <ul class="spaced-lists">
+          <li>Atom</li>
+          <li>Notepad++</li>
+          <li>Sublime</li>
+          <li>VS Code</li>
+          <li>CodeSandBox</li>
+        </ul>
+    </div>
+    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C0%29&t=one-dark&wt=none&l=htmlmixed&width=800&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=6px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522spaced-lists%2522%253E%250A%2520%2520%253Cli%253EAtom%253C%252Fli%253E%250A%2520%2520%253Cli%253ENotepad%252B%252B%253C%252Fli%253E%250A%2520%2520%253Cli%253ESublime%253C%252Fli%253E%250A%2520%2520%253Cli%253EVS%2520Code%253C%252Fli%253E%250A%2520%2520%253Cli%253ECodeSandBox%253C%252Fli%253E%250A%253C%252Ful%253E%250A"
+  style="width: 800px; height: 318px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+  <h2 class="pb">Stacked Lists</h2>
+  <div class="list-container">
+    <ul class="stacked-lists">
+      <li>Atom</li>
+      <li>Notepad++</li>
+      <li>Sublime</li>
+      <li>VS Code</li>
+      <li>CodeSandBox</li>
+    </ul>
+</div>
+<iframe
+src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C0%29&t=one-dark&wt=none&l=htmlmixed&width=800&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=6px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cul%2520class%253D%2522stacked-lists%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253EAtom%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253ENotepad%252B%252B%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253ESublime%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253EVS%2520Code%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253Cli%253ECodeSandBox%253C%252Fli%253E%250A%253C%252Ful%253E"
+style="width: 800px; height: 299px; border:0; transform: scale(1); overflow:hidden;"
+sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+  <h2 class="pb">Navigation List</h2>
+    <div class="list-container">
+      <nav class="nav-vertical-list">
+      <ul class="link-list">
+      <li class="nav-list"><a class="btn btn-link page-link"><div>Home</div></a></li>
```

Comment:
Nice that you have shown list with links.

Reply:
Thank you